Libplanet.Net.Tests/BoundPeerTest.cs

```diff
@@ -32,28 +30,12 @@ public class BoundPeerTest
 
         [Theory]
         [MemberData(nameof(GetBoundPeers))]
-        public void Serializable(BoundPeer peer)
+        public void Bencode(BoundPeer peer)
         {
-            var formatter = new BinaryFormatter();
-            using (var stream = new MemoryStream())
-            {
-                formatter.Serialize(stream, peer);
-                byte[] serialized = stream.ToArray();
-                stream.Seek(0, SeekOrigin.Begin);
-                BoundPeer deserialized = (BoundPeer)formatter.Deserialize(stream);
-                Assert.IsType(peer.GetType(), deserialized);
-                Assert.Equal(peer, deserialized);
-            }
-        }
-
-        [Theory]
-        [MemberData(nameof(GetBoundPeers))]
-        public void Serialize(BoundPeer peer)
-        {
-            Bencodex.Types.Dictionary serialized = peer.ToBencodex();
-            var deserialized = new BoundPeer(serialized);
+            Bencodex.Types.IValue bencoded = peer.Bencoded;
+            var decoded = new BoundPeer(bencoded);
 
-            Assert.Equal(peer, deserialized);
+            Assert.Equal(peer, decoded);
```

Comment:
Maybe it might be beneficial to have a `NotEqual` case.

Reply:
I'm not quite getting it, what would be a `NotEqual()` case?

Reply:
I was thinking of something like creating a `BoundPeer` that has roughly the default value, and doing `Assert.NotEqual()` with it, but now that I think of it the idea of a default `PublicKey` might be not well-defined. 🤔 I take it back.